packages/demand-bloc-trip-create/package.json

```diff
@@ -24,8 +24,9 @@
   "dependencies": {
     "@karhoo/demand-api": "^11.6.0",
     "camelcase-keys": "^6.2.2",
+    "dayjs": "^1.10.4",
     "rxjs": "6.5.5",
-    "yup": "^0.32.8"
+    "yup": "^1.0.0-alpha.2"
   },
```

Comment:
Why did you decide to use alpha version?

Reply:
There is an issue on `yup`'s side that prevents building the package :( https://github.com/jquense/yup/pull/1204

Reply:
AFAIU the main reason is to not spend time on moving from `0.32.8` to `1.x.x` version in the future. It might have lots of breaking changes